README.md

```diff
@@ -6,40 +6,40 @@ User application for Agoric Inter Protocol--Vaults, BLD Boost, Liquidations, etc
 
 1. Download and build the latest copy of `agoric-sdk`.
 
-```
-cd agoric-sdk
-yarn && yarn build
-```
+   ```
+   cd agoric-sdk
+   yarn && yarn build
+   ```
 
 2. (One-time) Define keys expected by the startup scripts
 
-```
-agd keys add gov1 --keyring-backend=test
-agd keys add gov2 --keyring-backend=test
-```
+   ```
+   agd keys add gov1 --keyring-backend=test
+   agd keys add gov2 --keyring-backend=test
+   ```
 
-If you have access to the mnemonics, use `--interactive` to get a prompt to import them. Without that it will create new keys. You can save the seed phrase somewhere, or even create a keplr account with it for testing. You can use `agd keys list --keyring-backend=test` to check which keys you've created.
+   If you have access to the mnemonics, use `--interactive` to get a prompt to import them. Without that it will create new keys. You can save the seed phrase somewhere, or even create a keplr account with it for testing. You can use `agd keys list --keyring-backend=test` to check which keys you've created.
 
 3. Start a local chain with psm, vaults, etc.
 
-```
-cd packages/inter-protocol && scripts/start-local-chain.sh
-```
+   ```
+   cd packages/inter-protocol && scripts/start-local-chain.sh
+   ```
 
-4. Start a local [wallet](https://github.com/Agoric/wallet-app) client server:
+4. Go the the [wallet](https://github.com/Agoric/wallet-app) repository and start a local client server:
 
-```
-cd wallet-app
-yarn start
-```
+   ```
+   cd wallet-app
```

Comment:
This needs to be updated to `cd wallet-app && yarn install` then `cd wallet && yarn start`

Reply:
Done.